src/classes/Cookbook.js

```diff
@@ -0,0 +1,39 @@
+const Recipe = require('../data/recipes.js');
+
+class Cookbook {
+  constructor(recipes) {
+    this.recipes = recipes;
+
+    // One class to get you started!
+  }
+  filterTags(selectedTag) {
+    const filteredRecipesByTag = this.recipes.filter(recipe => {
+      return recipe.tags.includes(selectedTag)
+})
+    return filteredRecipesByTag
+  }
+  // filteredIngredients(searchedIngredient) {
```

Comment:
I still think this might be beneficial moving forward... just need to flesh out what exactly this can do for us moving forward